app/js/lib/reducers/input_validation.js

```diff
@@ -22,6 +22,19 @@ function inputValidation( validationState, action ) {
 				isValid: inputIsValid( action.payload.value, action.payload.pattern )
 			};
 			return newValidationState;
+		case 'VALIDATE_FIELDSET':
```

Comment:
I'd rather call this `MARK_EMPTY_FIELD_AS_INVALID` or `MARK_UNCHANGED_FIELDS_AS_INVALID` or something, because no real validation is taking place here.